### ISSUE TITLE
Webview2 crashes when closing multiple windows

### ISSUE THREAD

**Issue by sudongg** [microsoft/microsoft-ui-xaml#7336]
### Describe the bug

In a WinAppSdk self-contained project, when multiple windows are open, and the windows contain a WebView2, it crashes on closing.
The solution is to use WebView.Close() to close the WebView in the Closed event of the main window.

### Steps to reproduce the bug

1. Create a new WinAppSdk project.
2.MainPage.xaml Add:

```
    <ContentPresenter x:Name="ContentPresenter" />
        <StackPanel Orientation="Vertical"  VerticalAlignment="Center" HorizontalAlignment="Left">
            <Button Click="Button_Click" Background="Red" Foreground="White" Content="New Window" Width="150" Height="50"/>
        </StackPanel>
```
3.MainPage.xaml.cs Add:

```
        public MainPage()
        {
            this.InitializeComponent();

            WebView2 WebView = new WebView2() { Source = new Uri("https://github.com/") };
            ContentPresenter.Content = WebView;
        }

        private void Button_Click(object sender, RoutedEventArgs e)
        {
            ContentPresenter.Content = null;
            MultiWindow appWindow = new MultiWindow(WebView);
            appWindow.Activate();
        }
```
4. Click the "New Window" button.
5. Close the window, crash。

### Expected behavior

won't crash

### NuGet package version

1.1.1

### Packaging type

Unpackaged

### Windows version

Windows 10 version 2004 (19041, May 2020 Update)

### IDE

Visual Studio 2022

**Comment by torum** [comment]
The solution "to use WebView.Close() to close the WebView in the Closed event of the main window" does not work when closing a child window containing WebView2 if it is behind the main window. It is strange but it crashes every time. 

Win11, WindowsAppSDK v1.2.230313.1, both packaged & unpackaged(self-contained)

**Comment by DmitriyKomin** [comment]
Duplicate of #6872 , fixed in Windows App SDK 1.3.1 (1.3.230502000)[](https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.3.1).

**Comment by DmitriyKomin** [comment]
Closing as it is fixed.
